analysis_scripts/analyze_quote_data/location-analysis_all_years.md

```diff
@@ -0,0 +1,234 @@
+                       bm_loc_df)
+bm_full_loc_df = subset(bm_full_loc_df, est_un_region != "" & 
+                                        est_un_subregion != "" &
+                                        est_un_region != "NO_EST" & 
+                                        est_un_subregion != "NO_EST")
+```
+
+### compare benchmark and non-benchmark data
+
+Now lets check how well the benchmark proportions match the full dataset proportions. The number of articles per year in each benchmark set is 10, the total number of articles in the year is between 400-1000. Since the number of articles in the benchmark set is very small, we don't expect exact consistency between the benchmark and total data sets.
+
+#### Lets look at the largest region definition first: UN region
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-4-1.png" style="display: block; margin: auto;" />
+
+#### Lets look at the UN Subregion level
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-5-1.png" style="display: block; margin: auto;" /><img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-5-2.png" style="display: block; margin: auto;" />
+
+#### Now Finally the Country Code level, except divided by subregion
+
+    ## $`1`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-1.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`2`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-2.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`3`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-3.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`4`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-4.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`5`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-5.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`6`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-6.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`7`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-7.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## $`8`
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-6-8.png" style="display: block; margin: auto;" />
+
+    ## 
+    ## attr(,"class")
+    ## [1] "list"      "ggarrange"
+
+### Compare locations over all years
+
+Now lets see the proportion of articles that mention each UN Subregion.
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-7-1.png" style="display: block; margin: auto;" /><img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-7-2.png" style="display: block; margin: auto;" />
```

Comment:
I might be missing something here: I see in the code that you are grouping by `is_benchmark`, but not sure which plot comes from the benchmark and the full data.

Reply:
I'm not sure how to read this plot: are you comparing fractions between the plot above and the one below? If so, why is "Northern America" not present in the plot below? Sorry if I'm missing something obvious (I have that feeling haha).

Reply:
No worries, this section isn't well documented and a bit confusing. So I am grouping on `is_benchmark`, but this is only so the column is not lost so I can re-use the method `get_aggr_region`, which expects a column named `is_benchmark`. In the end there is only 1 unique element in the `is_benchmark` column.

For the two plots, they are on the same data: the full data set. The only difference is that the bottom excludes Northern America + Northern Europe because I wanted to better visualize the other subregions with lower proportions. I added some extra description in 39ebb1e to hopefully make it a bit clearer.